Formula/pdnsrec.rb

```diff
@@ -3,7 +3,7 @@ class Pdnsrec < Formula
   homepage "https://www.powerdns.com/recursor.html"
   url "https://downloads.powerdns.com/releases/pdns-recursor-4.7.3.tar.bz2"
   sha256 "206d766cc8f0189f79d69af64d8d937ecc61a4d13e8ea6594d78fe30e61405f2"
-  license "GPL-2.0-only"
+  license "GPL-2.0-only" => { with: "openvpn-openssl-exception" }
```

Comment:
Exact phrasing from NOTICE:
```
In addition, for the avoidance of any doubt, permission is granted to
link this program with OpenSSL and to (re)distribute the binaries
produced as the result of such linking.
```

https://spdx.org/licenses/openvpn-openssl-exception.html is the only SPDX OpenSSL exception right now.